apps/fishing-map/features/workspace/highlight-panel/highlight-panel.content.ts

```diff
@@ -4,7 +4,7 @@ import { Locale } from 'types'
 // Update with a new id when a new release
 // This id is used for highlighting the dataview with a popup on the right
 // update it here if you want to show it again
-export const HIGHLIGHT_DATAVIEW_INSTANCE_ID = 'highlight-sar-match'
+export const HIGHLIGHT_DATAVIEW_INSTANCE_ID = 'vms'
```

Comment:
You should use this dataviewInstanceId in the instance to highlight, so in this case you have to remove from sar and use it [here](https://github.com/GlobalFishingWatch/frontend/blob/ff90e7dac686c818fa72c3f81cde8506dbea3dd1/apps/fishing-map/data/default-workspaces/workspace.development.ts#L59)